src/rtcModule/webrtc.cpp

```diff
@@ -1976,16 +1976,17 @@ void Call::destroyIfNoSessionsOrRetries(TermCode reason)
     }
 
     auto wptr = weakHandle();
-    auto wptrManager = mManager.weakHandle();
-    mManager.mRetryCallTimers[chatid] = setTimeout([this, wptr, wptrManager, chatid, reason]()
+    RtcModule* manager = &mManager;
+    auto wptrManager = manager->weakHandle();
+    mManager.mRetryCallTimers[chatid] = setTimeout([this, wptr, wptrManager, manager, chatid, reason]()
```

Comment:
Should we additionally specify the copy-ctor as deleted for `RtcModule`? That way, this issue would be caught at compile time.

Reply:
Nevermind, I see now that your aim was to keep the pointer.